from PyQt5 import QtCore, QtGui, QtWidgets

class Ui_MainWindow(object):
    def setupUi(self, MainWindow):
        MainWindow.setObjectName("MainWindow")
        MainWindow.resize(640, 480)
        self.centralwidget = QtWidgets.QWidget(MainWindow)
        self.centralwidget.setObjectName("centralwidget")
        self.listWidget = QtWidgets.QListWidget(self.centralwidget)
        self.listWidget.setGeometry(QtCore.QRect(450, 130, 171, 141))
        self.listWidget.setObjectName("listWidget")
        self.listWidget_2 = QtWidgets.QListWidget(self.centralwidget)
        self.listWidget_2.setGeometry(QtCore.QRect(450, 280, 171, 141))
        self.listWidget_2.setObjectName("listWidget_2")
        self.Rules = QtWidgets.QListWidget(self.centralwidget)
        self.Rules.setGeometry(QtCore.QRect(15, 21, 421, 401))
        self.Rules.setObjectName("Rules")
        self.groupBox = QtWidgets.QGroupBox(self.centralwidget)
        self.groupBox.setGeometry(QtCore.QRect(450, 10, 171, 114))
        self.groupBox.setObjectName("groupBox")
        self.verticalLayout = QtWidgets.QVBoxLayout(self.groupBox)
        self.verticalLayout.setObjectName("verticalLayout")
        self.pushButton = QtWidgets.QPushButton(self.groupBox)
        self.pushButton.setObjectName("pushButton")
        self.verticalLayout.addWidget(self.pushButton)
        self.pushButton_2 = QtWidgets.QPushButton(self.groupBox)
        self.pushButton_2.setObjectName("pushButton_2")
        self.verticalLayout.addWidget(self.pushButton_2)
        self.pushButton_3 = QtWidgets.QPushButton(self.groupBox)
        self.pushButton_3.setObjectName("pushButton_3")
        self.verticalLayout.addWidget(self.pushButton_3)
        MainWindow.setCentralWidget(self.centralwidget)
        self.menubar = QtWidgets.QMenuBar(MainWindow)
        self.menubar.setGeometry(QtCore.QRect(0, 0, 640, 21))
        self.menubar.setObjectName("menubar")
        self.menuFile = QtWidgets.QMenu(self.menubar)
        self.menuFile.setObjectName("menuFile")
        self.menuKnowledge = QtWidgets.QMenu(self.menubar)
        self.menuKnowledge.setObjectName("menuKnowledge")
        self.menuRecommendation = QtWidgets.QMenu(self.menubar)
        self.menuRecommendation.setObjectName("menuRecommendation")
        self.menuSolution = QtWidgets.QMenu(self.menubar)
        self.menuSolution.setObjectName("menuSolution")
        MainWindow.setMenuBar(self.menubar)
        self.statusbar = QtWidgets.QStatusBar(MainWindow)
        self.statusbar.setObjectName("statusbar")
        MainWindow.setStatusBar(self.statusbar)
        self.actionOpen = QtWidgets.QAction(MainWindow)
        self.actionOpen.setObjectName("actionOpen")
        self.actionSave = QtWidgets.QAction(MainWindow)
        self.actionSave.setObjectName("actionSave")
        self.actionNew = QtWidgets.QAction(MainWindow)
        self.actionNew.setObjectName("actionNew")
        self.actionExit = QtWidgets.QAction(MainWindow)
        self.actionExit.setObjectName("actionExit")
        self.actionVariables = QtWidgets.QAction(MainWindow)
        self.actionVariables.setObjectName("actionVariables")
        self.actionDomains = QtWidgets.QAction(MainWindow)
        self.actionDomains.setObjectName("actionDomains")
        self.actionStart = QtWidgets.QAction(MainWindow)
        self.actionStart.setObjectName("actionStart")
        self.actionViewOntology = QtWidgets.QAction(MainWindow)
        self.actionViewOntology.setObjectName("actionViewOntology")
        self.actionSolution = QtWidgets.QAction(MainWindow)
        self.actionSolution.setObjectName("actionSolution")
        self.menuFile.addAction(self.actionNew)
        self.menuFile.addAction(self.actionOpen)
        self.menuFile.addAction(self.actionSave)
        self.menuFile.addSeparator()
        self.menuFile.addAction(self.actionExit)
        self.menuKnowledge.addAction(self.actionVariables)
        self.menuKnowledge.addAction(self.actionDomains)
        self.menuKnowledge.addAction(self.actionViewOntology)
        self.menuRecommendation.addAction(self.actionStart)
        self.menuSolution.addAction(self.actionSolution)
        self.menubar.addAction(self.menuFile.menuAction())
        self.menubar.addAction(self.menuKnowledge.menuAction())
        self.menubar.addAction(self.menuRecommendation.menuAction())
        self.menubar.addAction(self.menuSolution.menuAction())

        self.retranslateUi(MainWindow)
        QtCore.QMetaObject.connectSlotsByName(MainWindow)

    def retranslateUi(self, MainWindow):
        _translate = QtCore.QCoreApplication.translate
        MainWindow.setWindowTitle(_translate("MainWindow", "MainWindow"))
        self.groupBox.setTitle(_translate("MainWindow", "Действия"))
        self.pushButton.setText(_translate("MainWindow", "Добавить"))
        self.pushButton_2.setText(_translate("MainWindow", "Редактировать"))
        self.pushButton_3.setText(_translate("MainWindow", "Удалить"))
        self.menuFile.setTitle(_translate("MainWindow", "Файл"))
        self.menuKnowledge.setTitle(_translate("MainWindow", "Знания"))
        self.menuRecommendation.setTitle(_translate("MainWindow", "Рекомендация"))
        self.menuSolution.setTitle(_translate("MainWindow", "Решение"))
        self.actionOpen.setText(_translate("MainWindow", "Открыть ..."))
        self.actionOpen.setShortcut(_translate("MainWindow", "Ctrl+O"))
        self.actionSave.setText(_translate("MainWindow", "Сохранить как ..."))
        self.actionSave.setShortcut(_translate("MainWindow", "Ctrl+S"))
        self.actionNew.setText(_translate("MainWindow", "Создать ..."))
        self.actionNew.setShortcut(_translate("MainWindow", "Ctrl+N"))
        self.actionExit.setText(_translate("MainWindow", "Выход"))
        self.actionExit.setShortcut(_translate("MainWindow", "Ctrl+Q"))
        self.actionVariables.setText(_translate("MainWindow", "Переменные"))
        self.actionVariables.setShortcut(_translate("MainWindow", "Ctrl+V"))
        self.actionDomains.setText(_translate("MainWindow", "Домены"))
        self.actionDomains.setShortcut(_translate("MainWindow", "Ctrl+D"))
        self.actionStart.setText(_translate("MainWindow", "Начать"))
        self.actionStart.setShortcut(_translate("MainWindow", "Ctrl+S"))
        self.actionViewOntology.setText(_translate("MainWindow", "Просмотр отнологии"))
        self.actionViewOntology.setShortcut(_translate("MainWindow", "Ctrl+V"))
        self.actionSolution.setText(_translate("MainWindow", "Показать решение"))